keyboards/fearherbs1/blue_team_pad/readme.md

```diff
@@ -0,0 +1,30 @@
+# Blue Team Pad
+
+![blue_team_pad](https://i.imgur.com/U8LNofv.jpeg)
```

Comment:
This uses a smaller, but still high res version of the image that is more suitable for github's markdown rendering.
```suggestion
![blue_team_pad](https://i.imgur.com/U8LNofvh.jpeg)
```